[RM Ch10] Gaps sections updated (#2695)
cntt-n/CNTT: doc/ref_model/chapters/chapter10.md

<a name="10.3.3"></a>
### 10.3.3 Closed-loop automation 
The state of a system is defined by a set of variables that fully describe the system and determines the response of the system to any given set of inputs. A closed loop automation system automatically maintains the specified desired state of the controlled system. 

Closed-loop automation is evolving as a major advancement in the telecommunication network automation. In the context of telecommunication systems, it means a system that in a continuous loop programmatically validates the state of the cloud infrastructure against the declared desired state, and in case of deviation from the desires state, it automatically takes remediation actions necessary for bringing the actual state to the desired state. The Reference Model specification will in its next releases address this important area.

<a name="10.3.64"></a>
### 10.3.4 Hybrid Multi-Cloud: APIs

> Section "8.2.4 Multi-Cloud Interactions Model" defines several core roles within the Multi-Cloud Model and discusses stereo-typical interactions between them. However, the Model realises that a federated cloud requires the definition and agreement on a set of APIs. Currently there exist in the industry various:

Reviewer: ```suggestion
Section "8.2.4 Multi-Cloud Interactions Model" defines several core roles within the Multi-Cloud Model and discusses stereo-typical interactions between them. However, the Model realises that a federated cloud requires the definition and agreement on a set of APIs. The current fragmentation in the industry is caused by various factors:
```
